test/performance/odp_l2fwd.c

```diff
@@ -1574,6 +1877,50 @@ static void create_groups(int num, odp_schedule_group_t *group)
 	}
 }
 
+static int set_vector_pool_params(odp_pool_param_t *params, odp_pool_capability_t pool_capa)
+{
+	uint32_t num_vec, vec_size;
+
+	if (gbl_args->appl.vec_size == 0)
+		vec_size = DEFAULT_VEC_SIZE;
+	else
+		vec_size = gbl_args->appl.vec_size;
+
+	if (pool_capa.vector.max_size &&
```

Comment:
`pool_capa.vector.max_size` should be always greater than zero if vector pool is supported.

Reply:
Will change `pool_capa.vector.max_size` check to assert.
`ODPH_ASSERT(pool_capa.vector.max_size > 0);`